implants/lib/eldritch/src/assets/copy_impl.rs

```diff
@@ -71,7 +79,26 @@ pub fn copy(starlark_eval: &Evaluator<'_, '_>, src: String, dst: String) -> Resu
             return copy_remote(rx, dst);
         }
     }
-    copy_local(src, dst)
+
+    let local_assets = starlark_eval.module().get("local_assets");
+    if let Some(assets) = local_assets {
+        let tmp_list = ListRef::from_value(assets).context("`local_assets` is not type list")?;
+        let src_value = starlark_eval.module().heap().alloc_str(&src);
+
+        if let Some(assets_dir_value) = starlark_eval.module().get("local_assets_directory") {
+            // Attempt to convert the Starlark Value to a Rust string
+            let local_assets_dir = assets_dir_value
+                .unpack_str()
+                .context("`local_assets_directory` is not a string")?;
+            if tmp_list.contains(&src_value.to_value()) {
```

Comment:
Should probably add a note to the docs that by design local file assets will only expose files that exist at start and files created in that dir during execution won't be available to the `assets` lib which is by desgin.